src/components/form/RadioButton.module.css

```diff
@@ -0,0 +1,47 @@
+@import '@digdir/design-system-tokens/brand/digdir/tokens.css';
```

Comment:
As we already import the altinn brand tokens, will this affect the global tokens? 🤔 What's the reason we have to import the digdir branding here - I'd at least want to include a comment about it so that we can consider when we can remove it safely again (i.e. when this works with the Altinn branding).

Reply:
Seems like we don't have to import the digdir branding here. Just a mistake from my part :sweat_smile: